File: main/src/main/java/net/fwitz/math/main/binary/split/functions/examples/simple/SplitIdentityMain.java
package net.fwitz.math.main.binary.split.functions.examples.simple;

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitIdentityMain {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("z")
                .computeFn(z -> z)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/examples/simple/SplitIdentityMain.kt
package net.fwitz.math.main.binary.split.functions.examples.simple

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitIdentityMain {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("z")
        .computeFn { z -> z }
        .render()
}